moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_display.cpp

```diff
@@ -142,6 +144,16 @@ MotionPlanningDisplay::MotionPlanningDisplay()
                               plan_category_, SLOT(changedQueryMarkerScale()), this);
   query_marker_scale_property_->setMin(0.0f);
 
+  query_marker_offset_category_ = new rviz::Property("Interactive Marker Offset", QVariant(), "", plan_category_, nullptr, this);
+  query_marker_offset_position_property_ =
+      new rviz::VectorProperty("Position", Ogre::Vector3(0, 0, 0),
+                               "Specifies the interactive marker offset position.",
+                               query_marker_offset_category_, SLOT(changedQueryMarkerOffset()), this);
+  query_marker_offset_orientation_property_ =
+      new rviz::QuaternionProperty("Orientation", Ogre::Quaternion(1, 0, 0, 0),
+                               "Specifies the interactive marker offset orientation.",
+                               query_marker_offset_category_, SLOT(changedQueryMarkerOffset()), this);
+
```

Comment:
This is just me being bothered by how QT works.  But I don't like the fact that these are class variables and we allocate them using new and then don't handle deleting them when this class is destroyed if they still exist.  If the lifetime of them is managed by some qt object we hand them to, maybe they should be local variables instead of class variables.

What you are doing is unrelated to the code smell of this file, carry on.